tests/test_potentials.py

```diff
@@ -33,20 +34,76 @@ def test_bound_potential_get_potential(harmonic_bond):
 
 def test_summed_potential_raises_on_inconsistent_lengths(harmonic_bond):
     with pytest.raises(ValueError) as excinfo:
-        potentials.SummedPotential([harmonic_bond], [])
+        SummedPotential([harmonic_bond], [])
 
     assert str(excinfo.value) == "number of potentials != number of parameter arrays"
 
 
 def test_summed_potential_keeps_referenced_potentials_alive(harmonic_bond):
-    sp = potentials.SummedPotential([harmonic_bond], [harmonic_bond.params]).bind(harmonic_bond.params)
+    sp = SummedPotential([harmonic_bond], [harmonic_bond.params]).bind(harmonic_bond.params)
 
     # segfaults if referenced potentials have been deallocated prematurely
     execute_bound_impl(sp.bound_impl(np.float32))
 
 
 def test_summed_potential_get_potentials(harmonic_bond):
     impls = [harmonic_bond.unbound_impl(np.float32) for _ in range(2)]
-    params_sizes = [len(harmonic_bond.params) for _ in range(2)]
+    params_sizes = [harmonic_bond.params.size] * 2
```

Comment:
Good catch! This was intentional; `params_sizes` is supposed to contain the lengths of the flattened parameter arrays. Since the harmonic bond params have shape `(N, 2)`, this was incorrect before.

Reply:
Makes sense -- thanks for confirming!